sentry/src/main/java/io/sentry/Attachment.java

```diff
@@ -0,0 +1,109 @@
+package io.sentry;
+
+import java.io.File;
+import org.jetbrains.annotations.ApiStatus;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+/** You can use an attachment to store additional files alongside an event or transaction. */
+@ApiStatus.Experimental
+public final class Attachment {
+
+  private @Nullable byte[] bytes;
+  private @Nullable String path;
+  private final String filename;
+  private String contentType;
```

Comment:
M: if they are supposed to be NotNull, let's add an annotation as well, it's easier to handle on Kotlin

Reply:
Do you know if there is a linter for this? So you don't have to make this comment.

Reply:
no afaik, that's why I try to be the human linter about that, Kotlin fails compilation for that, but not on Java side.

Reply:
If you say Kotlin fails compilation for that. What do you exactly mean?

Reply:
if a Java method is annotated with @NotNull or @Nullable, when Kotlin code is run against those annotations, they use the null safety feature accordingly, so you can't pass a `null` value to a `@NotNull` Java method, unless you use reflection.
you should know that, as you've written tests using reflection :D

Reply:
Now I get your point. I thought you meant that Kotlin would fail if I didn't add the annotation. That would be news for me.